FrontEnd/src/UserAuthentication/LandingPage.jsx

```diff
@@ -39,11 +39,7 @@ function LandingPage() {
             </div>
 
             <div className="links">
-              <ul className={isMenuOpen ? "active" : ""}>
-                <li>
-                  <a href="#">About Us</a>
-                </li>
-              </ul>
+              <ul className={isMenuOpen ? "active" : ""}></ul>
```

Comment:
Which menu is this? I didn't see any on the landing page: https://serviceitt.netlify.app/

Reply:
It's for small screen sizes

Reply:
Incase any header texts bigger than screen width. But there's a very small chance of that happening since we no longer have "About us" page so I can take it out.